### PR TITLE
XX 149  | SFButton Migration

### PR DESCRIPTION
**Pull request by Yopo-Nicolas**

### CODE REVIEW

**Review by pdcastano — CHANGES_REQUESTED** [review]
- disabled style is wrong
- invertedGrey color styles look wrong.

File: src/Components/SFButton/CustomButtons/ButtonMixin.tsx [hunk 1]
```diff
@@ -69,138 +68,128 @@ export interface ButtonMixinProps {
 
 // TODO: Fix this 'any' return type
 export default function ButtonMixin(sfButtonConfig: ButtonMixinProps): any {
```
Comment (pdcastano) [inline src/Components/SFButton/CustomButtons/ButtonMixin.tsx:70]: Is there any way to fix this missed type?

**Review by pdcastano — CHANGES_REQUESTED** [review]
- In `outlined` and `contained` button variants the text is in uppercase

File: src/Components/SFButton/CustomButtons/ButtonMixin.tsx (hunk 1 above)
Comment (Yopo-Nicolas) [reply src/Components/SFButton/CustomButtons/ButtonMixin.tsx:70]: This return type is due to MUI styled function returns generic return types. It's done this way to style any component.

File: src/Components/SFButton/CustomButtons/ButtonMixin.tsx [hunk 2]
```diff
@@ -69,138 +68,130 @@ export interface ButtonMixinProps {
...
+    display: 'flex',
+    gap: '6px',
+    color: sfButtonConfig.contained[theme.palette.mode].color,
+    textTransform: 'unset',
+    backgroundColor:
+      sfButtonConfig.contained[theme.palette.mode].backgroundColor,
+    '@media (hover: hover)': {
+      '&:hover, &:focus': {
+        backgroundColor:
+          sfButtonConfig.contained[theme.palette.mode].backgroundColorHover
```
Comment (pdcastano) [inline src/Components/SFButton/CustomButtons/ButtonMixin.tsx:81]: `:focus` should be replaced with `':focus-visible` as we've defined in the FE meeting.

Comment (pdcastano) [reply src/Components/SFButton/CustomButtons/ButtonMixin.tsx:81]: Apply this in the whole file

**Review by pdcastano — CHANGES_REQUESTED** [review]

File: src/Components/SFButton/CustomButtons/ButtonMixin.tsx (hunk 2 above)
Comment (Yopo-Nicolas) [reply src/Components/SFButton/CustomButtons/ButtonMixin.tsx]: Done

**Review by pdcastano — APPROVED** [review]

**Comment by pdcastano** [comment]
LGTM

**Pull request merged**